internal/sail/sail.go

```diff
@@ -10,7 +10,7 @@ import (
 	"time"
 
 	"github.com/spf13/cobra"
-	hudServer "github.com/windmilleng/tilt/internal/hud/server"
+	assets2 "github.com/windmilleng/tilt/internal/assets"
```

Comment:
was this intentional or done by a refactor rename?

(it seems weird to rename the package rather than the var)